src/main/features/claim/routes/pay.ts

```diff
@@ -102,8 +74,30 @@ async function successHandler (res, next) {
       features += features === undefined ? 'directionsQuestionnaire' : ', directionsQuestionnaire'
     }
     await claimStoreClient.saveClaim(draft, user, features)
-  }
 
+  } catch (err) {
+    if (err.statusCode === HttpStatus.INTERNAL_SERVER_ERROR || err.statusCode === HttpStatus.SERVICE_UNAVAILABLE) {
+      logError(
+        user.id,
+        draft.document.claimant.payment,
+        `Payment processed successfully but there is problem saving claim from claim store externalId: ${externalId}.`
```

Comment:
Can we fix the grammar please?
`... there was a problem saving claim '${externalId}' to the claim store`

Reply:
Done @andrewwa-kainos .